.env.example

```diff
@@ -1,5 +1,13 @@
 TOKEN="MyBotToken"
 GUILD_ID=1234567890
+OWNERS="hello world"
+
 MODMAIL_GUILD_ID=1234567890
-OWNERS='hello world'
+
+# DATABASE
 CONNECTION_URI="mongodb+srv://mongodburi"
+
+# DISCORD LOGIN
+CLIENT_ID = "YOUR APP'S CLIENT ID HERE"
+CLIENT_SECRET = "YOUR APP'S CLIENT SECRET HERE"
+REDIRECT_URI = "YOUR DISD LOGIN REDIRECT URI"
```

Comment:
This should be `"https://yourdomain.com/callback"` so that /callback is not forgotten. That or we should construct it down where we use it, so you just put "yourdomain.com" as an example.